qa/pom.xml

```diff
@@ -126,19 +126,23 @@
 
     <dependency>
       <groupId>com.oracle.database.jdbc</groupId>
-      <artifactId>ojdbc8</artifactId>
+      <artifactId>ojdbc10</artifactId>
+      <version>19.21.0.0</version>
```

Comment:
Why have you introduced `<version>` here and not in the `narayana-bom-test`

Reply:
Thanks Manuel, good catch. I will update the bom and remove version from there. I will double check that we use that driver version everywhere in Narayana.